src/pages/item/item.html

```diff
@@ -15,28 +15,28 @@
       <ion-item>
 				<ion-label floating color="primary">Brand</ion-label>
 				<ion-select [(ngModel)]="item.brandID" name="brand">
-          <ion-option *ngFor="let brand of brands" value="{{brand.id}}">{{brand.name}}</ion-option>
+          <ion-option *ngFor="let brand of brands" value="{{brand.brandID}}">{{brand.name}}</ion-option>
```

Comment:
No, you actually need to use the brackets notation. Without it, the value is interpreted as a string.